meinberlin/apps/votes/models.py

```diff
@@ -56,14 +57,42 @@ def get_or_create_salt_for_module(module):
         return token_salt.salt
 
 
+class TokenPackage(models.Model):
+    module = models.ForeignKey(Module, on_delete=models.CASCADE)
+    size = models.PositiveIntegerField()
```

Comment:
Didnt we want to remove the size and only use num_tokens instead??

Reply:
> Didnt we want to remove the size and only use num_tokens instead?

Ah, but we need it to check if it can be downloaded already, right?

Reply:
yes, exactly, and to check if we reached the limit already